rcgen/Cargo.toml

```diff
@@ -0,0 +1,54 @@
+[package]
+name = "rcgen"
+version = "0.11.3"
+description = "Rust X.509 certificate generator"
+repository = "https://github.com/rustls/rcgen"
```

Comment:
can this be a workspace-level thingy as well?

Reply:
`name` cannot. I moved the rest in latest commit. I also moved `keywords` up a level. reference for keys allowed in `workspace.package`: https://doc.rust-lang.org/cargo/reference/workspaces.html#the-package-table